pkg/util/validation/limits.go

```diff
@@ -11,9 +11,11 @@ import (
 	"flag"
 	"fmt"
 	"math"
+	"reflect"
 	"strings"
 	"time"
 
+	"github.com/efficientgo/core/errors"
```

Comment:
Can we use stdlib or `pkg/errors` as everywhere else?

Reply:
OMG good catch, IDK what my IDE has imported here. I was hoping for our standard `errors.Wrap()`. Changed to `fmt.Errorf` in https://github.com/grafana/mimir/pull/4397/commits/1cdc036da7b6fac1ba82997de0ddb44b9e24ee95